src/contact.rs

```diff
@@ -1069,12 +1078,12 @@ impl Contact {
                 .sql
                 .query_map(
                     "SELECT c.id, c.addr FROM contacts c
-                 LEFT JOIN acpeerstates ps ON c.addr=ps.addr  \
                  WHERE c.id>?
+                 AND c.fingerprint!='' \
                  AND c.origin>=? \
                  AND c.blocked=0 \
                  AND (iif(c.name='',c.authname,c.name) LIKE ? OR c.addr LIKE ?) \
-                 AND (1=? OR LENGTH(ps.verified_key_fingerprint)!=0)  \
+                 AND (1=? OR c.verifier!=0)  \
```

Comment:
There seems to be no test for `DC_GCL_VERIFIED_ONLY` (all tests pass if you replace this with `(1=? OR 1=1)`), and also, not sure whether anything uses this flag; maybe we can just remove it in a follow-up PR. Not sure whether we should add a test for it now.

When adding contacts to a group, Android offers all contacts and then shows a warning if you try to add a non-verified contact, so, at least Android doesn't use this flag.